subprojects/core/src/main/java/org/gradle/internal/operations/trace/BuildOperationRecord.java

```diff
@@ -18,13 +18,32 @@
 
 import com.google.common.base.Function;
 import com.google.common.collect.Lists;
+import com.google.common.collect.Ordering;
 
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 
 public final class BuildOperationRecord {
 
+    public static final Ordering<BuildOperationRecord> ORDERING = Ordering.natural()
```

Comment:
What's the reason behind adding this ordering?

Reply:
The sibling events are now ordered by who started first. Previously it was who finished first.